docs/adrs/002-change-release-strategy.md

```diff
@@ -0,0 +1,78 @@
+---
+status: proposed
+date: 17-05-2024
+deciders: Kamil Kołodziej, Radosław Szuma, Jakub Pisarek
+informed: Fishjam Team, Cloud Team
+---
+# Change release strategy to eliminate downtimes
+
+## Context and Problem Statement
+
+In the current situation when we do a release we are shuting down the server and every room with that.
+This is an awful user experience. Another issue is the fact that any ongoing recordings are lost.
+
+## Decision Drivers
+
+* N o downtime while release happens
+* 0 lost recordings during release
+
+## Considered Options
+
+* Rolling update with eviction
```

Comment:
Should we write more on the API? E.g. whether we want to add a new REST endpoint?